src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpRedirectFeaturesAnalyzers.cs

```diff
@@ -143,14 +145,16 @@ await TestServices.Workspace.WaitForAllAsyncOperationsAsync(
                 FeatureAttribute.ErrorList
              ],
              cancellationToken);
-        return await TestServices.ErrorList.GetErrorsAsync(cancellationToken);
+        return await TestServices.ErrorList.GetErrorsAsync(ErrorSource.Other, __VSERRORCATEGORY.EC_WARNING, cancellationToken);
```

Comment:
❔ Why this change? None of the tests build the project, so `ErrorSource.Build` (which this change explictly omits) is guaranteed to be empty. The default behavior of this call is a better match to the default and most common IDE configuration for users.